openedx/features/course_experience/views/course_updates.py

```diff
@@ -48,17 +51,29 @@ def render_to_fragment(self, request, course_id=None, **kwargs):
         course_url_name = default_course_url_name(request)
         course_url = reverse(course_url_name, kwargs={'course_id': unicode(course.id)})
 
-        # Fetch the updates as HTML
-        updates_html = get_course_info_section(request, request.user, course, 'updates')
+        # Fetch all of the updates
+        info_module = get_course_info_section_module(request, request.user, course, 'updates')
+        ordered_updates = self.order_updates(info_module.items)
 
         # Render the course home fragment
         context = {
             'csrf': csrf(request)['csrf_token'],
             'course': course,
             'course_url': course_url,
-            'updates_html': updates_html,
+            'updates': ordered_updates,
             'disable_courseware_js': True,
             'uses_pattern_library': True,
         }
         html = render_to_string('course_experience/course-updates-fragment.html', context)
         return Fragment(html)
+
+    def order_updates(self, updates):
```

Comment:
With the new fragment class, there no longer is a global CourseInfoModule object on the page. When trying to call the ordered_updates from here we get 

`TypeError: unbound method ordered_updates() must be called with CourseInfoModule instance as first argument (got CourseUpdatesFragmentView instance instead)`

It will save us the overhead of avoiding needing the entire info object if we move the function over and make it static, as you recommended, so that it can be safely added without conflicting.